api/v1alpha1/api.go

```diff
@@ -106,6 +106,10 @@ type LLMBackendSpec struct {
 	//
 	// +kubebuilder:validation:Required
 	BackendRef egv1a1.BackendRef `json:"backendRef"`
+
+	// LLMSecurityPolicyName list the LLMSecurityPolicy that this backend will depend on
+	//
+	LLMSecurityPolicyName *string `json:"llmSecurityPolicyName,omitempty"`
```

Comment:
```suggestion
	securityPolicyName *string `json:"securityPolicyName,omitempty"`
```

Reply:
I've updated this, but I'm worried this will get confused with the envoy gateway security policy